src/server/api/API_ingest/updated_data.py

```diff
@@ -14,21 +14,21 @@ def get_updated_contact_data():
     select json_agg (upd)  as "cd"
     from (
         select
-        sf.source_id as "Id" ,  -- long salesforce string
-        array_agg(sl.source_id) filter (where sl.source_id is not null)   as  "Person_Id__c",           -- short PAWS-local shelterluv id
+        sf.source_id as "contactId" ,  -- long salesforce string
```

Comment:
I think this may be a more clear view when publishing the payloads. "SELECT 
    sf.source_id AS "Id",  -- long salesforce string
    array_agg(sl.source_id) FILTER (WHERE sl.source_id IS NOT NULL) AS "Person_Id__c",  -- short PAWS-local shelterluv id
    sf.source_id AS "contactId"  -- long salesforce string fix
FROM
    -- your_table_name_here -- add your table name here
GROUP BY 
    sf.source_id;"